### ISSUE TITLE
Place templates in .github/notify-slack-action

### ISSUE THREAD

**Issue by davidkarlsen** [evryfs/notify-slack-action#3]
Would it make more sense to keep the templates in a magic dir/file, rather than having to have a separate branch for templates? Then branching would work as normal. This technique is quite common and for instance used in the release-drafter action: https://github.com/release-drafter/release-drafter

**Comment by rainui28** [comment]
Would it be possible to use te release drafter on the `templates` branch or `master` with that configuration?
The idea of having `templates` as an orphan branch is to possess his:

1. personal timeline: it avoid conflicts when PR are created. 
2. lightweight branch: when contributors are cloning the project, they can clone only the `templates` branch by running `git clone https://github.com/evryfs/notify-slack-action.git -b templates --single-branch`. In fact, they won't have to pull the `node_modules` and source code which can be heavy, but only the templates JSON files with guideline.

In short, this practice looks like having two disctint repository within the same repo. I believe that we can still use the *release-drafter* by keeping this architecture. At least, *release-drafter* is a genius idea! :D

What do you think?

**Comment by rainui28** [comment]
ERRATUM:

3. Own release page: by having the orphan branch, the `master` *release* section will possess only the logic/feature code while the `templates` *release* section will only have the new add templates.

If you think about it, you can keep the current release `js-v1.0.0` that works fine while the `templates` branch is evolving. You can have `templates` release up to `v.5.0.0` for instance (ie. +100 templates) while the source code is still at `js-v1.0.0` release.
